prime-router/settings/organizations-local.yml

```diff
@@ -445,3 +448,17 @@
           type: REDOX
           apiKey: some_key
           baseUrl: http://redox:1080
+      - name: HL7_NULL
+        organizationName: ignore
```

Comment:
Added this new Receiver, to test using the NullTransport.    NullTransport does all the database stuff, but doesn't call SFTP or any transport.  Lets us distinguish between slow-downs caused by sftp vs slowdowns caused by lots of database interactions.